renderer/components/nettests/performance/Dash.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+  Flex,
+  Box,
+  Text
+} from 'ooni-components'
+import { FormattedMessage } from 'react-intl'
+
+import StatusBox from '../../measurement/StatusBox'
+import formatBitrate from '../../formatBitrate'
+import formatSpeed from '../../formatSpeed'
+import performanceTestGroup from './index'
+
+const Dash = ({measurement, render}) => {
+  const testKeys = JSON.parse(measurement.test_keys)
+  const {
+    connect_latency,
+    median_bitrate,
+    min_playout_delay
+  } = testKeys
+
+  // Get human readable description of the bitrate (e.g 4k, FHD, 240p)
+  const optimalQuality = formatBitrate(median_bitrate)
+  const bitrate = formatSpeed(median_bitrate)
+
+  const DashHero = (
+    <Box width={1} p={3}>
+      <Flex flexWrap='wrap' my={4} alignItems='center'>
+        <Box my={3}>
+          <FormattedMessage
+            id='TestResults.Details.Performance.Dash.VideoWithoutBuffering'
+            values={{
+              VideoQuality: optimalQuality
+            }}
+          />
+        </Box>
+      </Flex>
+    </Box>
+  )
+
+  const DashTitle = (
+    <FormattedMessage
+      id='TestResults.Details.Performance.Dash.VideoWithoutBuffering'
+      values={{
+        VideoQuality: optimalQuality
+      }}
+    />
+  )
+
+  const DashDetails = () => (
+    <Box width={1}>
+      <Flex flexWrap='wrap' alignItems='center' my={4}>
+        <Box width={1/2} my={4}>
+          <StatusBox
+            label={<FormattedMessage id='TestResults.Details.Performance.Dash.MedianBitrate' />}
+            value={<Text>{bitrate.value} <small>{bitrate.unit}</small></Text>}
+          />
+        </Box>
+        <Box width={1/2}>
+          <StatusBox
+            label={<FormattedMessage id='TestResults.Details.Performance.Dash.PlayoutDelay' />}
+            value={min_playout_delay}
+          />
+        </Box>
+      </Flex>
+    </Box>
+  )
+
+  return (
+    <div>
+      {render({
```

Comment:
why is `render` here? I seem to not understand where it is needed

Reply:
This is the pattern I was talking about in [the WebConnectivity PR](https://github.com/ooni/probe-desktop/pull/49#issuecomment-539690345). We receive `render` function from the parent component which constructs what is to be rendered here based on `details` from the child here.

Reply:
ah, gotcha!